app/src/main/java/com/codesoom/assignment/domain/ProductUpdateRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.codesoom.assignment.domain;
+
+
+/**
+ * 상품 변경에 필요한 데이터를 정의하는 인터페이스 입니다.
+ */
```

Comment:
단호한 주석을 작성하는 연습도 해봅시다.
```suggestion
/**
 * 상품 변경 요청.
 */
```

Reply:
주석 전체적으로 단호하게 변경해 봤습니다.